src/composables/deepLinkApi.ts

```diff
@@ -26,8 +28,11 @@ export function useDeepLinkApi({ router }: UseDeepLinkApiOptions) {
   /**
    * Function needed to support legacy tipping from superhero.com
    */
-  function checkIfOpenTransferSendModal() {
-    if (checkIfSuperheroCallbackUrl(route.query)) {
+  async function checkIfOpenTransferSendModal(currentRoute: Route) {
+    if (
+      currentRoute.path.slice(1) === ROUTE_ACCOUNT
```

Comment:
Wouldn't `currentRoute.name` resolve the need of using slice?

Reply:
`name` is `undefined` in this case.